lacework/casting.go

```diff
@@ -57,3 +131,12 @@ func joinMapStrings(m map[string]string, delimit string) string {
 	}
 	return strings.Join(out, delimit)
 }
+
+func ContainsStr(array []string, expected string) bool {
```

Comment:
If we're going to export this function could it be moved to an internal arrays "helpers" package like we do in go-sdk?

Reply:
I think we should make it public since now we are using it here too 😂 - I will do that